networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/GossipForkManager.java

```diff
@@ -188,6 +190,15 @@ public synchronized void publishDataColumnSidecar(final DataColumnSidecar dataCo
         GossipForkSubscriptions::publishDataColumnSidecar);
   }
 
+  public void publishExecutionProof(final ExecutionProof executionProof) {
+    // for now we don't have a slot in the message data (execution proof) to use
+    // I believe it's safe to just check the current epoch
+    LOG.debug("Publishing execution proof for slot {}", executionProof);
+    UInt64 slot = spec.computeStartSlotAtEpoch(spec.getCurrentEpoch(recentChainData.getStore()));
```

Comment:
Probably worth adding a TODO so we rememebr to revisit this later. Needs further discussion with Kev.